Frameworks/VisaCheckoutSDK.framework/Headers/VisaCheckoutButton.h

```diff
@@ -74,25 +75,19 @@ NS_SWIFT_NAME(onCheckout(total:currency:completion:));
 /** A value indicating whether the Visa Checkout SDK is configured and ready to launch,
  The VisaCheckoutButton will be enabled when this property is true (and disabled when this property is false).
  */
-@property (nonatomic, readonly) BOOL isReady;
+@property (nonatomic, readonly) BOOL isReady DEPRECATED_MSG_ATTRIBUTE("Please use `onReady` callback to know when Visa Checkout is ready to launch");
 
-/** Convenience property for Interface Builder to set the value of VisaCheckoutButtonStyleStandard.
- Default value is true.
- */
+/// :nodoc:
```

Comment:
It's not quite clear to me why docs are being removed on some of these properties and not others. Was it decided that some of the docs are no longer accurate/useful, or is the convention to remove docs on deprecated methods/properties?

Reply:
Ah, we should have called this out in the PR description, but everything under `Frameworks/VisaCheckoutSDK.framework` is owned by Visa Checkout. We copied over the new version of their framework, so that's why these changes show up in the diff. But these are Visa's changes, not ours.